app/controllers/api/v1/tasks_controller.rb

```diff
@@ -15,6 +15,17 @@ def update
 
         head :no_content
       end
+
+      private
+
+      def params_for_update
+        permitted = api_doc_definition.all_attributes - api_doc_definition.read_only_attributes
+        if body_params['context'].present?
+          permitted.delete('context')
+          permitted << { "context" => {} }
```

Comment:
Do you mind changing to no spaces inside the braces for consistency?

Reply:
do you think `permitted << {'context'=>{}}` ? I would be ok with it, but rubocop isn't.